app/models/check.rb

```diff
@@ -0,0 +1,11 @@
+class Check < ApplicationRecord
+  belongs_to :claim
+
+  enum result: {
+    approved: 0,
+  }
+
+  def readonly?
+    persisted?
+  end
+end
```

Comment:
Do we not need to validate the presence of `result` and `checked_by`?

Reply:
yep, we should do that 👍🏽